templates/pvc.yaml

```diff
@@ -53,17 +53,32 @@ spec:
   #storageClassName: {{ .Values.storage_class_rwmany }}
   #volumeMode: Filesystem
 
-{{ if .Values.extra_instr_dir.enabled | default false }}
+# {{ if .Values.extra_instr_dir.enabled | default false }}
+# ---
+# apiVersion: v1
+# kind: PersistentVolumeClaim
+# metadata:
+#   name: instruments-dir
+# spec:
+#   accessModes:
+#   - ReadWriteMany
+#   resources:
+#     requests:
+#       storage: 1G
+#   storageClassName: {{ .Values.storage_class_rwmany }}
+# {{ end }}
+
```

Comment:
It's commented here, but I still see a volume in prod, and it's actually mounted and used in frontend and bot. The PVC is declared separately in the flux configuration.

Was there a specific reason for this?